simple_template_server/README.md

```diff
@@ -0,0 +1,50 @@
+Template Development Server
+===================================
+
+This is a simple Snippet Server for development of Snippet Templates. It ignores
+the client configuration including locale, Firefox version and release channels
+and just returns all the snippets to the requesting client. If you only care for
+Snippet Template development this is the server for you.
+
+Installation
+------------
+
+1. Create and activate a Python virtualenv
+
+    ``` shell
+    virtualenv venv
+    . venv/bin/activate
+    ```
+
+2. Install the needed packages
+
+    ``` shell
+    pip install -r requirements.txt
```

Comment:
OK obviously the instructions are not clear, I need to fix that. There's a `requirements.txt` inside the `simple_template_server` directory. That's the one you need to install, not the global one.